fast/stages/1-resman/variables-fast.tf

```diff
@@ -68,6 +68,12 @@ variable "custom_roles" {
   default = null
 }
 
+variable "default_alerts_email" {
```

Comment:
this might be different for notifications set here, I would not include it in FAST variables